### ISSUE TITLE
finetuning on libero

### ISSUE THREAD

**Issue by lzl2040** [thu-ml/RoboticsDiffusionTransformer#102]
When I try to finetune RDT on LIBERO, the loss curve is as follows:

<img width="824" alt="Image" src="https://github.com/user-attachments/assets/f71fee25-6aea-4252-be87-279d98c60c8d" />
Is this normal? My dataset preparation is as follows:

- complete the ```libero_vla_dataset.py``` based on [hdf5_maniskill_dataset.py](https://github.com/thu-ml/RoboticsDiffusionTransformer/blob/main/data/hdf5_maniskill_dataset.py): Because the libero provides the actions and the  states in the format of ```x, y, z, roll, pitch, yaw, gripper```, I first transform it to [6D representation](https://arxiv.org/pdf/1812.07035). Then I use the following state:
```python
UNI_STATE_INDICES = [ STATE_VEC_IDX_MAPPING['eef_pos_x']
                            ] + [ STATE_VEC_IDX_MAPPING['eef_pos_y']
                            ] + [ STATE_VEC_IDX_MAPPING['eef_pos_z']
                    ] + [
                        STATE_VEC_IDX_MAPPING[f"eef_angle_{i}"] for i in range(6)
                    ] + [
                        STATE_VEC_IDX_MAPPING["right_gripper_open"]
                    ]
```
Is there any error?

**Comment by jasper0314** [comment]
Hi @lzl2040 and @Buming-O , could you please share the performance of RDT on LIBERO that you’ve tried? I’m considering using RDT, but I’m not sure how well it works in simulation environments like LIBERO or SimplerEnv. Thanks!

**Comment by Zctoylm0927** [comment]
> Hello! Thanks for your work. I'm also trying to finetuning LIBERO (using the original HDF5 version instead of the RLDS version provided by OpenVLA), and here are the current results. Can this be considered convergence?
> 
> ![Image](https://github.com/user-attachments/assets/547893fb-9210-41c1-9a21-d0d17f29b5c3)

Great try, could you please share the performance of RDT on LIBERO? I also encountered some problems during training.

**Comment by kkyywww** [comment]
May I ask, does the loss drop to around 10^-4 after a few dozen fine-tuning steps? @lzl2040 @csuastt

**Comment by RIP4KOBE** [comment]
Hi  @lzl2040 @Buming-O , thanks for sharing your attempt at fine-tuning RDT on LIBERO — this is a very useful direction for the community.

If possible, would you be willing to share the fine-tuned RDT checkpoint or any training artifacts/configs you used? That would be really helpful for others who want to reproduce or build upon your work, and could save repeated fine-tuning efforts.

Thanks again!
